js/lib/beautify-html.js

```diff
@@ -536,6 +545,31 @@
                 return comment;
             };
 
+            this.get_handlebars_comment = function(start_pos) { //function to return handlebars comment content in its entirety
```

Comment:
Shouldn't this be added to the comment handling method above instead of being made into a separate method?